plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -41,20 +41,9 @@ test.describe( 'Add New Simple Product Page', () => {
 			consumerSecret: process.env.CONSUMER_SECRET,
 			version: 'wc/v3',
 		} );
-		await api.get( 'products' ).then( ( response ) => {
-			const products = response.data;
-			for ( const product of products ) {
-				if (
-					product.name === virtualProductName ||
-					product.name === nonVirtualProductName
-				) {
-					api.delete( `products/${ product.id }`, {
-						force: true,
-					} ).then( () => {
-						// nothing to do here.
-					} );
-				}
-			}
+		await api.delete( `products/${ virtualProductId }`, { force: true } );
+		await api.delete( `products/${ nonVirtualProductId }`, {
+			force: true,
```

Comment:
The test became more stable when product ID's were determined early on in the test and then used here for deletion, rather than searching the entire list of products by name and then getting their id.